.github/workflows/ci.yml

```diff
@@ -256,6 +258,17 @@ jobs:
       project-name: OpenTelemetry.Instrumentation.Runtime
       code-cov-name: Instrumentation.Runtime
 
+  build-test-semanticconventions:
+    needs: detect-changes
+    if: |
+      contains(needs.detect-changes.outputs.changes, 'semanticconventions')
+      || contains(needs.detect-changes.outputs.changes, 'build')
+      || contains(needs.detect-changes.outputs.changes, 'shared')
+    uses: ./.github/workflows/Component.BuildTest.yml
+    with:
+      project-name: OpenTelemetry.SemanticConventions
+      code-cov-name: SemanticConventions
```

Comment:
@joegoldman2 Add this new `code-cov-name` definition (as `unittests-SemanticConventions`) in here: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/dc7be37862e4669638bb9df42db83794cbb667d7/.github/codecov.yml#L34

Reply:
I ended up pushing some changes to the branch to fix CI so I added this.